src/sass/pages/projects.scss

```diff
@@ -1,5 +1,43 @@
 .projects-hero {
-  // FIXME: this is just a placeholder until there's something in this page
   @include corporate-gradient;
-  height: 600px;
+  padding: ($padding-hero-vertical + $height-topbar + $height-secondary-nav) 0 0 0;
+  overflow: hidden;
+
+  &__wrap {
+    @include row;
+    @include container;
+    position: relative;
+
+    .code-pills-left {
+      position: absolute;
+      // this position is so that the code pills match with the position of the
+      // pill in the title
+      right: 90%;
+      top: -69px;
```

Comment:
I will probably spend the next sprint reviewing CSS and changing all px to rems. I'll tell you more on planning, but I'll leave this here for now.